src/subscribers/longpoll.c

```diff
@@ -524,6 +541,42 @@ static void request_cleanup_handler(subscriber_t *sub) {
 }
 
 
+static void keep_alive_handler(ngx_event_t *ev) {
+  full_subscriber_t *fsub = (full_subscriber_t *)ev->data;
+
+  if(ev->timedout) {
+    ngx_buf_t   *buf = REQUEST_PCALLOC(fsub->sub.request, buf);
```

Comment:
doing a pool allocation here resutls in a momory leak over the lifetime of the subscriber request, as this memory is not released until the request is terminated.